5.5/root/usr/bin/run-mysqld-slave

```diff
@@ -2,23 +2,26 @@
 #
 # This is an entrypoint that runs the MySQL server in the 'slave' mode.
 #
-source ${HOME}/common.sh
+source /usr/share/container-scripts/mysql/common.sh
```

Comment:
This variable is defined in this file. We can replace it if we define it directly in the Dockerfile, which seems like a good idae btw. Do you think so as well?